src/config.js

```diff
@@ -1,6 +1,6 @@
 const local = {
-    mongoURI: 'mongodb://127.0.0.1:27017,127.0.0.1:27018,127.0.0.1:27019/poker-db?' +
-        'replicaSet=rs',
+    mongoURI: 'mongodb://megapokerbot:mega123@cluster0-shard-00-00.vpwpq.mongodb.net:27017,cluster0-shard-00-01.vpwpq.mongodb.net:27017,cluster0-shard-00-02.vpwpq.mongodb.net:27017/poker-db?' +
+        'replicaSet=atlas-hugr40-shard-0&ssl=true&authSource=admin',
```

Comment:
This is actually the production database URI. It is mentioned in the mongoURI environment variable in the YAML files. I don't think it is a good idea to have the development DB the same as the production DB.

Reply:
reverted back to the local URI